akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -369,4 +373,17 @@ object Source {
     new Source(new AcknowledgeSource(bufferSize, overflowStrategy, DefaultAttributes.acknowledgeSource, shape("AcknowledgeSource")))
   }
 
+  /**
+   * Creates a Source from a Files contents.
+   * Emitted elements are `chunkSize` sized [[akka.util.ByteString]] elements,
+   * except the final element, which will be up to `chunkSize` in size.
+   *
+   * You can configure the default dispatcher for this Source by changing the `akka.stream.blocking-io-dispatcher` or
+   * set it for a given Source by using [[ActorAttributes]].
+   *
+   * It materializes a [[Future]] containing the number of bytes read from the source file upon completion.
+   */
+  def file(f: File, chunkSize: Int = 8192): Source[ByteString, Future[Long]] =
```

Comment:
I don't think taking away the default is helping... 8192 is a default used in JDK impls as well as other IO things in scala (i.e. infameous io.Source). When picking this value I benchmarked various settings and this is pretty good as a default I found (can look for benchmarks if you're interested, but then again - it's jsut a default)